<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtcReceiveDlg</class>
 <widget class="QWidget" name="BtcReceiveDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receive Bitcoin</string>
  </property>
  <widget class="QTableWidget" name="tableAddresses">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Account</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Balance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Involves watchonly</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonNewAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate new address</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkShowEmpty">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Show empty</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkShowWatchonly">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show watchonly addresses and include them in balance calculations.</string>
   </property>
   <property name="text">
    <string>Show watchonly</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>581</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced:</string>
   </property>
   <widget class="QPlainTextEdit" name="textPublicKeys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>571</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter one public key or address per line. Addresses only work if bitcoin-qt knows their public key.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>255</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>List of public keys (one per line):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>215</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Signatures required:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinSigsRequired">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCreateMultisig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Address will be added and imported to wallet so transactions show up as watchonly.</string>
    </property>
    <property name="text">
     <string>Create multisig address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editImportAddress">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>45</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>75</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Account:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editImportAccount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>95</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>watchonly</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkImportRescan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>75</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rescan</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>155</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Account:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMultisAccount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>175</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>multisig</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonImport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Import watchonly address</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>275</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Account:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editAccount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>moneychanger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>340</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNewAddress">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>checkShowEmpty</zorder>
  <zorder>checkShowWatchonly</zorder>
  <zorder>groupBox</zorder>
  <zorder>buttonNewAddress</zorder>
  <zorder>tableAddresses</zorder>
  <zorder>label_3</zorder>
  <zorder>editAccount</zorder>
  <zorder>label_7</zorder>
  <zorder>editNewAddress</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
